### ISSUE TITLE
I'd like cursor history (back/forward) to work across files

### ISSUE THREAD

**Issue by rohrlich** [goki/gide#232]

**Comment by rcoreilly** [comment]
?  you mean it would switch to a different file if in your history you switched files at that point?  or what?  if that, it would definitely require a different mechanism -- currently it is equivalent to the emacs version of this feature.

**Comment by rohrlich** [comment]
Yes, that is what I mean. I know it isn't a simple extension of what you have. Goland and xCode have it and it is very nice when working with a bunch of files in the same package. Not that it cares about the packages in the project, just that it is most useful then.
